docs/api-reference/tasks-api.md

```diff
@@ -25,77 +25,1448 @@ sidebar_label: Tasks
+        "sqlQueryContext": {
+            "sqlQueryId": "32663269-ead9-405a-8eb6-0817a952ef47",
+            "sqlInsertSegmentGranularity": "\"DAY\"",
+            "maxNumTasks": 3,
+            "queryId": "32663269-ead9-405a-8eb6-0817a952ef47"
+        },
+        "sqlResultsContext": {
+            "timeZone": "UTC",
+            "serializeComplexValues": true,
+            "stringifyArrays": true
+        },
+        "sqlTypeNames": [
+            "TIMESTAMP",
+            "BIGINT",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "BIGINT",
+            "VARCHAR",
+            "VARCHAR",
+            "BIGINT",
+            "BIGINT",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR",
+            "VARCHAR"
+        ],
+        "context": {
+            "forceTimeChunkLock": true,
+            "useLineageBasedSegmentAllocation": true
+        },
+        "groupId": "query-32663269-ead9-405a-8eb6-0817a952ef47",
+        "dataSource": "wikipedia_api",
+        "resource": {
+            "availabilityGroup": "query-32663269-ead9-405a-8eb6-0817a952ef47",
+            "requiredCapacity": 1
+        }
+    }
+  }
+  ```
+
+</details>
+
+### Get task status
+
+#### URL
+<code class="getAPI">GET</code> `/druid/indexer/v1/task/{taskId}/status`
+
+Retrieves the status of a task given the task id. It returns a JSON object with the task's current status (`running`, `complete`, `waiting`, `pending`), task type, datasource, and other relevant metadata.
```

Comment:
We should refer to it as "state" instead of status to match the query parameter used in the first endpoint. Suggest also removing the parenthetical altogether, or creating a separate follow up sentence for it.